Application/EdFi.Ods.Features/OpenApiMetadata/Strategies/FactoryStrategies/OpenApiMetadataCompositePathsFactoryStrategy.cs

```diff
@@ -17,107 +17,111 @@
+                                                           {
+                                                               CompositeResourceContext =
                                                                         r.CompositeResourceContext,
-                                                                    Readable = true, Path =
-                                                                        $"/{_swaggerDocumentContext.CompositeContext.CategoryName}/{r.Resource.PluralName}",
-                                                                    RequestProperties = GetMetaDataPropertiesForCurrentResource(r)
-                                                                })
+                                                               Readable = true,
+                                                               Path =
+                                                                        $"/{_openApiMetadataDocumentContext.CompositeContext.CategoryName}/{r.Resource.PluralName}",
+                                                               RequestProperties = GetMetaDataPropertiesForCurrentResource(r)
+                                                           })
                                                       .ToList();
 
-            return swaggerResourceList
+            return openApiMetadataResourceList
                .Concat(
-                    swaggerResourceList
+                    openApiMetadataResourceList
                        .Where(r => r.IsCompositeResource)
                        .SelectMany(GetContainedResources));
         }
 
         public bool HasTotalCount => false;
 
-        private IEnumerable<SwaggerPathsResource> GetContainedResources(
-            SwaggerResource swaggerResource)
+        private IEnumerable<OpenApiMetadataPathsResource> GetContainedResources(
+            OpenApiMetadataResource openApiMetadataResource)
         {
-            return _swaggerDocumentContext.CompositeContext.RouteElements
+            return _openApiMetadataDocumentContext.CompositeContext.RouteElements
                                           .Where(
                                                route =>
                                                    !IsDefaultRoute(route)
-                                                   && RouteAppliesToResource(route, swaggerResource))
+                                                   && RouteAppliesToResource(route, openApiMetadataResource))
                                           .SelectMany(
                                                d =>
                                                {
-                                                   var operationSpecSuffix = GetOperationSpecNickname(swaggerResource.Resource, d);
+                                                   var operationSpecSuffix = GetOperationSpecNickname(openApiMetadataResource.Resource, d);
```

Comment:
lets fix the formatting on this method please

Reply:
Fixed